DataFormats/L1TParticleFlow/interface/HPSPFTau.h

```diff
@@ -135,11 +199,12 @@ namespace l1t {
     bool passVLooseRelIso_;
   };
 
-  void printPFCand(ostream& os, const l1t::PFCandidate& l1PFCand, const l1t::TkPrimaryVertexRef& primaryVertex);
-  void printPFCand(ostream& os, const l1t::PFCandidate& l1PFCand, float primaryVertex_z);
 }  // namespace l1t
 
 /// print to stream
 std::ostream& operator<<(std::ostream& os, const l1t::HPSPFTau& l1PFTau);
 
+void printPFCand(ostream& os, const l1t::PFCandidate& l1PFCand, const l1t::TkPrimaryVertexRef& primaryVertex);
+void printPFCand(ostream& os, const l1t::PFCandidate& l1PFCand, float primaryVertexZ);
+
```

Comment:
A curiosity, why did you move these functions out of the l1t namespace and into the global namespace?